### PR TITLE
{bio,vis}[foss/2022a] n2v v0.3.2, CSBDeep v0.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3130.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/c3e94d266dc2619584470c00bec8464b for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/n/n2v/n2v-0.3.2-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'PythonBundle'
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/n/n2v/n2v-0.3.2-foss-2022a.eb:1]: I might have missed a memo :), are we using `PythonBundle` even for a single python package?

Comment (Micket) [reply easybuild/easyconfigs/n/n2v/n2v-0.3.2-foss-2022a.eb:1]: Actually yes. I think we said it was less messing about in those cases where we need to add or remove deps, from just add/remove a few lines to completely restructure the easyconfig. But we haven't used PythonPackage as a reason to hold anything back from a merge.

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

**Pull request merged**